### DISCUSSION TITLE
Selamat Datang di Forum Diskusi hidrokit!

### DISCUSSION THREAD

**Discussion by taruma** [hidrokit/hidrokit#123] [Umum]
## 👋 Selamat Datang!
Kami menggunakan forum diskusi ini sebagai tempat untuk menghubungkan antara anggota dalam komunitas kami. Kami harap:
- Bertanya soal apapun.
- Berbagi ide.
- Berinteraksi antara anggota komunitas.
- Menyambut siapapun dan berpikiran terbuka. Ingat, bahwa komunitas ini kita bangun bersama. 

Untuk memulainya, ayo beri komentar dibawah dengan memperkenalkan diri anda dan ceritakan apa rencanamu di dalam komunitas ini. 

-----

**Comment by taruma** [comment]
Halo semuanya, perkenalkan nama saya Taruma, atau bisa dipanggil uma. Saya merupakan lulusan teknik sipil, kebetulan skripsi dan tesis terkait bidang teknik sumberdaya air. Akan tetapi, saya masih sangat awam dalam wawasan proyek teknik sumberdaya air. Jadi, jangan aneh yah kalau saya banyak nanya. Hehe. 

Saya memulai proyek hidrokit untuk mengembangkan proyek _open-source_ python yang bisa digunakan untuk wadah kolaborasi dan menjembatani antara para praktisi/akademisi di bidang sumberdaya air dengan informatika. Saya harap dengan proyeknya ini akan memunculkan proyek lainnya yang mengutamakan kolaborasi dan komunikasi demi mencapai tujuan yang sama.

Sekian dulu perkenalan dari saya, untuk lebih jauhnya bisa kunjungi profil saya @taruma di github.

**Comment by iingLK** [comment]
Hai nama saya iing. Senang berkenalan dengan yang ada disini. Sya terbiasa menggunakan command prompt dan comand line, penglaman membuat program menggunakan bahasa pascal compiler delphi 15 tahun yang lalu.
